src/components/Layout/FullBleed.tsx

```diff
@@ -0,0 +1,15 @@
+import { cloneElement, ReactElement } from "react";
+import { useScrollableParent } from "src/components/Layout/ScrollableParent";
+import { Css } from "src/Css";
+
+/** Provides a way to extend the full width of the ScrollableParent */
+export function FullBleed({ children, omitPadding = false }: { children: ReactElement; omitPadding?: boolean }) {
+  const { paddingX } = useScrollableParent();
+  return !paddingX
+    ? children
+    : cloneElement(children, {
+        style: Css.mxPx(paddingX * -1)
```

Comment:
I'm kinda surprised setting the `style` like this works...but I get it! The `.$` just returns a hash of CSS styles, so :shrug: it doesn't literally have to be output by emotion.